src/native/external/libunwind-version.txt

```diff
@@ -1,10 +1,6 @@
-v1.6.2
-https://github.com/libunwind/libunwind/commit/b3ca1b59a795a617877c01fe5d299ab7a07ff29d
+main@e07b43c02d5cf1ea060c018fdf2e2ad34b7c7d80
+https://github.com/libunwind/libunwind/commit/e07b43c02d5cf1ea060c018fdf2e2ad34b7c7d80
```

Comment:
This has pulled up a lot of unnecessary changes. An alternative fix is to keep pointing to released version 1.6.2, and apply patch from PR 330, as we have done for the entire loongarch64 support.

Reply:
The changes that it has pulled  in all look ok and simple. Besides the loongarch64 changes that are now included in the main and the PR 330, there were only five PRs - one just a licence header update, one adding couple of ifdefs for freebsd, one fixing malloc issue, one fixing a race and one updating the latex doc.

Reply:
Loongarch and  other patches were applied on top of 1.6.2 without resetting to libunwind's master branch, which is still using version 1.6.1-rc1. My comment was we can apply PR 330 in similar way which fixes the issue.

If you prefer this approach, ok. I think it just makes the versioning a bit ambiguous, but not a big deal.